experimental/uefi/runtime/src/framing.rs

```diff
@@ -14,26 +14,13 @@
 // limitations under the License.
 //
 
-extern crate alloc;
-
 use crate::{remote_attestation::AttestationHandler, Channel};
 use alloc::vec::Vec;
+use anyhow::Context;
 use ciborium::{de, ser};
 
-#[derive(Debug)]
-pub enum Error<T> {
-    // An error occured while deserializing.
-    Deserialization(ciborium::de::Error<T>),
-
-    // An error occured while serializing.
-    Serialization(ciborium::ser::Error<T>),
-
-    // An error occured in remote attestation.
-    Attestation(anyhow::Error),
-}
-
```

Comment:
One thing I liked about this struct is facilitating differentiation between types of errors that may either be private or could be forwarded to the client. We can properly tackle that as part of #2783 though, for now using anyhow consistently seems nice. :)

Reply:
I agree with the idea about deciding which error types should be reported to the client, but I don't think this enum provided a meaningful distinction. I agree that this should be addressed in a more systematic way as part of #2783 and #2794.